README-DEV.md

```diff
@@ -1,10 +1,37 @@
-Testing
--------
+# Testing
+
 Use `pytest` to run the tests in [the tests directory](qurator/dinglehopper/tests):
```

Comment:
I actually like the other style 😬 However, it's not _that_ important to me, so I'll merge it as is.

Reply:
Was not changing the style intentionally, sry about that (makes note to check Markdown styles before changing Markdown files ;)